install.ps1

```diff
@@ -840,6 +841,42 @@ foreach ($env in $configXml.config.envs.env) {
 }
 refreshenv
 
+Write-Host "[+] Checking for custom Start Layout file..."
+$customLayoutPath = Join-Path $desktopPath "CustomStartLayout.xml"
+if ([string]::IsNullOrEmpty($customLayout)) {
```

Comment:
The code we use to download `config.xml` and `CustomStartLayout.xml` is the same but using different url and filename. What about creating a helper function in this file (something like `Get-Config-File` with `$filePath` and `$url` as arguments) and use it for both functions? This way we don't have duplicated code and it should be easy to maintain the code.

Reply:
Completed. 👍